dynamicio/mixins.py

```diff
@@ -672,28 +676,64 @@ def _read_from_postgres(self) -> pd.DataFrame:
 
         connection_string = f"postgresql://{db_user}:{db_password}@{db_host}:{db_port}/{db_name}"
 
-        model = self.options.get("model")
         sql_query = self.options.get("sql_query")
 
         query = None
 
-        if model and sql_query:
-            raise ValueError("Only one of `model` and `sql_query` must be provided")
-        if model is None and sql_query is None:
-            raise ValueError("One of `model` and `sql_query` must be provided")
+        if self.options.get("model"):
+            raise ValueError("`model` must not be provided")
```

Comment:
`if model and sql_query:` --> correctly removed. 

`if model is None and sql_query is None:` --> correctly removed. 

`if self.options.get("model"):` --> we should remove this too. No need to tell users that this is no longer how `dynamicio` works. If one provides this `kwarg` it will be ignored and they will get a warning about it. 

Note that this is not a breaking change. Repositories that use dynamicio  with a model kwarg will still work fine with this enhancement (with the exception of warnings). 

Now, interacting with postgres can obviously be done in two ways: 
1. With a data model (which in this PR we are implicitly generating from the schema);
2. Through the provision of an sql query. 

Default behaviour is 1.  I see we are accounting for 2 conditionally if an `sql_query` is present. So we are good here.

Reply:
Great!